### DISCUSSION TITLE
Prebuilt virtual machines not opening

### DISCUSSION THREAD

**Discussion by Adiy458** [utmapp/UTM#5540] [UTM Help (iOS)] [answered]
When I press open on a prebuilt .utm file, it doesn’t open and nothing happens. (I used an android vm from the official page on GitHub)

* UTM Version: UTM SE 4.3.5 build 87
* OS Version: iOS 17 public beta
* Device Model: iPhone 11
* Is it jailbroken (name jailbreak used)? N/a
* How did you install UTM? Via scarlet 

No debug log as machine won’t upload 

config.plist file: https://drive.google.com/file/d/1e7emCxRvuAEciS_sIqyouDIn3lwRWqKe/view?usp=drivesdk

thanks !

**Comment by osy** [comment, marked as answer]
That’s not a supposed VM. Download one from https://mac.getutm.app/gallery/

also how did you press “open”. Where are you pressing that? Provide a screen recording if possible.

**Reply by Adiy458** [reply]
Is there any way to get a prebuilt android vm that is official?

**Reply by Adiy458** [reply]
Sorry, the certificate of the app got revoked:( I can't open it now will reply when unrevoked
